webapp/components/connectWallets/connectWalletsDrawer.tsx

```diff
@@ -47,9 +47,7 @@ export const ConnectWalletsDrawer = function ({ closeDrawer }: Props) {
       <div className="pb-18 h-full bg-white px-4 pt-6 md:max-w-md md:p-6 md:px-6">
```

Comment:
This is not part of your PR, but couldn't it just be simplified with `md:p-6` only?
```suggestion
      <div className="pb-18 h-full bg-white px-4 pt-6 md:max-w-md md:p-6">
```

Reply:
I'm going to review it and update the fix (if needed) as part of the following PR, so I don't block this one that has all the approvals 😅 .